### PR TITLE
Improved search bar styling

### PR DESCRIPTION
**Pull request by kirualex**
Testing a little PR

### CODE REVIEW

File: app/styles/components/Header.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Header
+
+.Header {
+
```
Comment (amilajack) [inline app/styles/components/Header.scss]: In this file, the spacing is a bit off. In your editor, try adding support for .editorconfig's. http://editorconfig.org/

**Comment by amilajack** [comment]
The convention of the project is to extend bootstrap's base configuration using SASS variables instead of css overrides. Try seeing if you can do this with bootstraps variables. The new variables you add should be place in variables.scss. You can find bootstrap's SASS variables in `node_modules/bootstrap/scss/_variables.scss`

File: app/styles/components/Header.scss (hunk 1 above)
Comment (thangngoc89) [inline app/styles/components/Header.scss]: Using stylelint could save you time explaining this kind of stuff

**Comment by amilajack** [comment]
Closing due to no response

**Pull request closed without merging**